editions/2015-06.md

```diff
@@ -0,0 +1,33 @@
+Ahoy there and welcome to the fifth edition of Civic Tech Monthly. Below you’ll find news and notes about civic tech from Australia and around the world.
```

Comment:
Aww, it's supposed to be a throwback to our first issue :)

I'm happy to drop it if you don't like it.